internal/services/oracledatabase/cloud_vm_cluster_resource.go

```diff
@@ -0,0 +1,429 @@
+// Copyright © 2024, Oracle and/or its affiliates. All rights reserved
+
+package oracledatabase
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2024-06-01/cloudvmclusters"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/tags"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+var _ sdk.Resource = CloudVmClusterResource{}
+
+type CloudVmClusterResource struct{}
+
+type CloudVmClusterResourceModel struct {
+	// Azure
+	Location          string                 `tfschema:"location"`
+	Name              string                 `tfschema:"name"`
+	ResourceGroupName string                 `tfschema:"resource_group_name"`
+	Tags              map[string]interface{} `tfschema:"tags"`
+
+	// Required
+	CloudExadataInfrastructureId string   `tfschema:"cloud_exadata_infrastructure_id"`
+	CpuCoreCount                 int64    `tfschema:"cpu_core_count"`
+	DataStorageSizeInTbs         float64  `tfschema:"data_storage_size_in_tbs"`
+	DbNodeStorageSizeInGbs       int64    `tfschema:"db_node_storage_size_in_gbs"`
+	DbServers                    []string `tfschema:"db_servers"`
+	DisplayName                  string   `tfschema:"display_name"`
+	GiVersion                    string   `tfschema:"gi_version"`
+	Hostname                     string   `tfschema:"hostname"`
+	LicenseModel                 string   `tfschema:"license_model"`
+	MemorySizeInGbs              int64    `tfschema:"memory_size_in_gbs"`
+	SshPublicKeys                []string `tfschema:"ssh_public_keys"`
+	SubnetId                     string   `tfschema:"subnet_id"`
+	VnetId                       string   `tfschema:"vnet_id"`
+
+	// Optional
+	BackupSubnetCidr         string                       `tfschema:"backup_subnet_cidr"`
+	ClusterName              string                       `tfschema:"cluster_name"`
+	DataCollectionOptions    []DataCollectionOptionsModel `tfschema:"data_collection_options"`
+	DataStoragePercentage    int64                        `tfschema:"data_storage_percentage"`
+	IsLocalBackupEnabled     bool                         `tfschema:"is_local_backup_enabled"`
+	IsSparseDiskgroupEnabled bool                         `tfschema:"is_sparse_diskgroup_enabled"`
+	TimeZone                 string                       `tfschema:"time_zone"`
+}
+
+func (CloudVmClusterResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		// Azure
+		"location": commonschema.Location(),
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+		},
+		"resource_group_name": {
```

Comment:
```suggestion
		"location": commonschema.Location(),
		
		"name": {
			Type:     pluginsdk.TypeString,
			Required: true,
		},
		
		"resource_group_name": {
```
should be new lines here. apply to all other schema properties

Reply:
I've added new lines for all the schema properties.